### ISSUE TITLE
Jaws of Life can't perform the "Remove the electronics from the airlock assembly step"

### ISSUE THREAD

**Issue by CydiaLamiales** [BeeStation/BeeStation-Hornet#7939]
## Reproduction:
Whether it be deconstructing and airlock or repairing an emagged one, Jaws of Life on the "prying jaws" mode cannot perform the "Remove airlock electronics" step while a crowbar can.
![image](https://user-images.githubusercontent.com/110184118/196862248-e5f430f1-c778-4cea-b99f-e872cc9f74af.png)
